weblate/accounts/tests/test_views.py

```diff
@@ -515,7 +515,7 @@ def test_watch(self) -> None:
         # Mute notifications for project
         self.client.post(reverse("mute", kwargs={"path": self.project.get_url_path()}))
         self.assertEqual(
-            self.user.subscription_set.filter(project=self.project).count(), 18
+            self.user.subscription_set.filter(project=self.project).count(), 20
```

Comment:
There is also failure above:

```
======================================================================
FAIL: test_watch (weblate.accounts.tests.test_views.ProfileTest.test_watch)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/weblate/weblate/weblate/accounts/tests/test_views.py", line 511, in test_watch
    self.assertEqual(
AssertionError: 20 != 18
```

See https://github.com/WeblateOrg/weblate/actions/runs/9573747546/job/26395752876?pr=11712